nmpolicy/internal/parser/parser_test.go

```diff
@@ -36,6 +36,12 @@ func TestParser(t *testing.T) {
 
 func testParseFailures(t *testing.T) {
 	var tests = []test{
+		{
```

Comment:
Why did you add the test to `testParseFailures`? I doesn't fail, just returns am empty ast.
BTW, I think we should fail and not allow an empty expression. But I guess now the failure will happen in the resolver, right?

Reply:
Right, moving to Success, yes.